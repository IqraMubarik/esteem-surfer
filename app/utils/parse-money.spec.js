/* eslint-disable */

import parseMoney from './parse-money';

describe('parse money', () => {
    it('(1) should parse', () => {
        const input = '18.494 SBD';
        expect(parseMoney(input)).toMatchSnapshot();
    });

    it('(2) should parse', () => {
        const input = '0.000 STEEM';
        expect(parseMoney(input)).toMatchSnapshot();
    });
});
