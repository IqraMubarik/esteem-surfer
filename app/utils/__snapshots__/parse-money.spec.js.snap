// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse money (1) should parse 1`] = `18.494`;

exports[`parse money (2) should parse 1`] = `0`;
